<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frameSetting">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>361</width>
     <height>571</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>770</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>201</width>
     <height>571</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QPushButton" name="pushButtonTransfer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxBrightness">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1023</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set all in this pattern</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonON">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShowIO">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Show I/O</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEditVersion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPatternNo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No 0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 7</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pattern No</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPatternComment">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxManual">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual control</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Operation</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pattern No</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPatternNoOperation">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No 0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>No 7</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Light No</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLightNo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Light 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Light 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Light 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Light 4</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditLightComment">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
